entries/QUnit.module.xml

```diff
@@ -36,13 +42,13 @@
 			If <code>QUnit.module</code> is defined without a <code>nested</code> callback argument, all subsequently defined tests will be grouped into the module until another module is defined.
 		</p>
 		<p>
-			Modules with grouped test functions allow defining nested modules, and QUnit will run tests on the parent module before going deep on the nested ones, even if they're declared first. The <code>beforeEach</code> and <code>afterEach</code> callbacks on a nested module call will stack (<a href="https://en.wikipedia.org/wiki/Stack_%28abstract_data_type%29">LIFO - last in, first out</a>) to the parent hooks.
+			Modules with grouped test functions allow defining nested modules, and QUnit will run tests on the parent module before going deep on the nested ones, even if they're declared first. Any hook callbacks on a nested module call will stack (<a href="https://en.wikipedia.org/wiki/Stack_%28abstract_data_type%29">LIFO - last in, first out</a>) to the parent hooks.
 		</p>
 		<p>
-			You can specify code to run before and after each test using the hooks argument, and also to create properties that will be shared on the context of each test. Any additional properties on the <code>hooks</code> object will be added to that context. The <code>hooks</code> argument is still optional if you call <code>QUnit.module</code> with a callback argument.
+			You can specify code to run before and after tests using the hooks argument, and also to create properties that will be shared on the testing context. Any additional properties on the <code>hooks</code> object will be added to that context. The <code>hooks</code> argument is still optional if you call <code>QUnit.module</code> with a callback argument.
 		</p>
 		<p>
-			The module's callback is invoked with the test environment as its <code>this</code> context, with the environment's properties copied to the module's tests, hooks, and nested modules. Note that changes on tests' <code>this</code> won't affect sibling tests, where <code>this</code> will be reset to the same value for each test.
+			The module's callback is invoked with the test environment as its <code>this</code> context, with the environment's properties copied to the module's tests, hooks, and nested modules. Note that changes on tests' <code>this</code> are not preserved between sibling tests, except when made in the <code>before</code> hook. In all other cases, <code>this</code> will be reset to the initial value for each test.
```

Comment:
`except when made in the <code>before</code> hook.` The phrase is already a reference for changes on each test not affecting sibling tests, the `before` hook is not one of them.